src/main/resources/config.yml

```diff
@@ -148,6 +148,7 @@ clan:
     max-description-length: 120
     min-description-length: 10
     max-members: 25
+    verified-max-members: 25

```

Comment:
This should be unverified-max-members, so it will interfere less with existing configs.

Reply:
You mean max-members should be unverified?

Reply:
Nope.

```yml
max-members: 25 # for verified clans
unverified-max-members: 10 #for unverified clans
```

This way existing verified clans won't be affected by this change.

Reply:
Ah okay, I get your point, will do asap